### ISSUE TITLE
You can't open the application "Electrum" because it is not supported on this type of Mac.

### ISSUE THREAD

**Issue by SomberNight** [spesmilo/electrum#5503]
Originally reported on [reddit](https://www.reddit.com/r/Electrum/comments/cbgb1q/problem_accessingdownloading_electrum_wallets_on/).

> I have been using Electrum wallets for the last couple years, and find it frustrating when they fail to open. A couple times a year I'd try to open my wallet on my Mac and get a message 'can't open Electrum, its not supported on this type of Mac.' Usually I find a fix by upgrading my Macbook version or upgrading to latest Electrum wallets. This time nothing works. I have deleted all previous electrum files (except for wallets/subwallets) and tried reinstalling Electrum multiple times, but keep getting a broken icon displaying the white circle/line), and the annoying message saying 'can't download Electrum as it's not supported on this type of Mac.' I have the latest Macbook version 10.14.5. Anyone else have similar issue opening Electrum (BTC and LTC) wallets, any workaround suggestions?

<img width="756" alt="picture of error" src="https://user-images.githubusercontent.com/29142493/60987376-012d9400-a331-11e9-82c2-67340b013398.png">

**Comment by BlackPelican78** [comment]
I have the latest MacOs Mojave version 10.14.5 installed. Have tried numerous downloads, even earlier Electrum wallet versions, but no luck downloading ('not supported on this type of Mac'). Any suggestions?

**Comment by SomberNight** [comment]
Have you seen this thread? https://discussions.apple.com/thread/4607662?page=2
Maybe try running it as another user; or from safe mode. Just to see if that works, not as a proper fix.

**Comment by BlackPelican78** [comment]
After restarting the Mac in safe mode, I was able to download the Electrum wallets (BTC & LTC), and both are fully functional. Thanks SomberNight for the suggestion, at least a temporary fix brings some relief. When restarting the Mac in regular mode, both wallets remain inoperable (same issue as reported earlier, will not open or download). What could explain this?

**Comment by danuker** [comment]
**TL; DR:** non-technical political rant about Apple's motives being aligned with this bug

This thread strikes me as not very coincidentally appearing at almost the same time as one of Apple's "carefully measured steps into bitcoin and crypto", as per [this article](https://www.forbes.com/sites/billybambrough/2019/06/06/apples-iphone-is-taking-its-first-steps-into-bitcoin-and-crypto/#355b02c520c6).

They intend to control you, as usual with nonfree software. Please read [Richard Stallman's notes](https://stallman.org/apple.html) on the subject. There even was [an older "ban"](https://www.theguardian.com/technology/2013/dec/10/apple-blocks-bitcoin-payments-on-secure-messaging-app-gliph) in force, but not this invasive.

In order to smother competition, it's supposed to be Embrace, Extend, Extinguish, but I think they started the "Extinguish" part a little too early.

**Comment by willkriski** [comment]
I have the same issue. I haven't opened Electrum in quite a few months and now I get that error. I can't install a new version either (unsupported). I'll try safe mode but this is scary for owning bitcoin if you can't open the wallet and there's no iOS app.

**Comment by danuker** [comment]
@willkriski Try a Linux live CD to back up your keys on a different device.

**Comment by willkriski** [comment]
i'm on mac but will have to learn how i can use the keys with different wallets

**Comment by SomberNight** [comment]
What kind of hardware do you have? What version of MacOS?
Do you get the same error when you try to start `/System/Library/CoreServices/Installer.app`?

**Comment by willkriski** [comment]
imac 2009. high sierra 10.13.6
how do you run that, it says it's a directory. The MacOS subdirectory is empty if that matters.

**Comment by SomberNight** [comment]
> how do you run that, it says it's a directory

Well it's a .app, you from the GUI (e.g. Finder), you can just double-click it. From terminal, run this instead:
`/System/Library/CoreServices/Installer.app/Contents/MacOS/Installer`
(just to see if it starts or gives same error)

**Comment by SomberNight** [comment]
> I haven't opened Electrum in quite a few months and now I get that error.

So the same version of Electrum that at some point worked is now also giving this error?

If so, what have you done in the meantime? Have you upgraded MacOS itself?

**Comment by willkriski** [comment]
it might have upgraded since then I'm not sure. I do some coding so possibly xcode upgrades, python, etc. 

when I run that new command you gave it opens a finder window, not sure what to do from there.

**Comment by SomberNight** [comment]
> it opens a finder window, not sure what to do from there.

Ok, so the Installer is not giving the same error...

Do you have some kind of antivirus software installed? Could that be silently blocking Electrum?

**Comment by willkriski** [comment]
that command is not running electrum just opening a finder window, no antivirus. how do i run electrum with your command as it's not specified?

**Comment by SomberNight** [comment]
That command is not intended to run Electrum. I was trying to figure out if you have a widespread issue on your Mac that affects many applications, or if it's only about Electrum. In the [above linked thread](https://discussions.apple.com/thread/4607662?page=2), the reporter cannot even start `Installer.app`.

**Comment by willkriski** [comment]
ah okay, i can run many apps electrum is the issue, hence no access to bitcoin which is troubling to say the least.

**Comment by SomberNight** [comment]
Well given you are a developer, you can always just run from source. :)

**Comment by willkriski** [comment]
i might have to but my expertise is in webmethods integration projects. assuming running from source would magically make it supported :)

**Comment by SomberNight** [comment]
Ok, another thing to try:
Right-click `Electrum.app` (file visible on screencap in OP) > `Show Package Contents`
There, start `Contents/MacOS/Electrum` (via double-clicking)

**Comment by willkriski** [comment]
nothing in the macOS folder

**Comment by SomberNight** [comment]
> no antivirus
> nothing in the macOS folder

Well it really sounds like an AV is interfering with the files...

Try copying `Electrum.app` to `/Applications` (e.g. by drag-and-dropping), and then from terminal, run:
```
/Applications/Electrum.app/Contents/MacOS/Electrum
```

**Comment by BlackPelican78** [comment]
> ah okay, i can run many apps electrum is the issue, hence no access to bitcoin which is troubling to say the least.

Most certaininly! As much as I've enjoyed the experience of using the wallets, I've decided to no longer use Electrum until there's a stable fix (this is an ongoing issue over the past year at least, and workarounds are getting more difficult.) Not being able to access wallets is concerning, and anxiety inducing, and no one wants that.

**Comment by thehapax** [comment]
> imac 2009. high sierra 10.13.6
> how do you run that, it says it's a directory. The MacOS subdirectory is empty if that matters.

I'm on 10.13.6 as well . Managed to resolve the issue by installing all the dependencies and run from source. After installing dependencies, then installing the binary release worked fine and "unsupported error" went away.

Noticed this was an issue with a new user account, fresh OSX reinstall btw, and homebrew tools and any dependencies were not installed.
